### PR TITLE
Refactor constraints in stepper filter view and view controller

### PR DESCRIPTION
**Pull request by shredlocker**
# Why?

Stepper filter was not behaving as expected.

# What?

Change constraints to make stepper filter view attached to bottom sheet.

# Show me

### Before

stepper-filter

### After

stepper-filter-fix

### CODE REVIEW

File: Sources/StepperFilter/StepperFilterViewController.swift [hunk 1]
```diff
@@ -35,7 +35,11 @@ public class StepperFilterViewController: UIViewController, FilterContainerViewC
     public override func viewDidLoad() {
         super.viewDidLoad()
         view.addSubview(stepperFilterView)
-        stepperFilterView.fillInSuperview()
+        NSLayoutConstraint.activate([
+            stepperFilterView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            stepperFilterView.topAnchor.constraint(equalTo: view.topAnchor, constant: 148),
```
Comment (osanoj) [inline Sources/StepperFilter/StepperFilterViewController.swift]: I don't like that `148` magic value... does it work on iPhone 4S? 😛 

Is there no other way to position it without magic constants? Or calculate it based on height?

Comment (shredlocker) [reply Sources/StepperFilter/StepperFilterViewController.swift]: Haha, I'll figure something out! 😄

Comment (osanoj) [reply Sources/StepperFilter/StepperFilterViewController.swift]: `.veryLargeSpacing * 2`  😜

Comment (shredlocker) [reply Sources/StepperFilter/StepperFilterViewController.swift]: Looks like that works well 😄

**Pull request merged**